src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,18 @@
+package step2
+
+enum class Operator(
+    val symbol: String,
+    val invoke: (Int, Int) -> Int
+) {
+    ADD("+", { target: Int, input: Int -> target + input }),
+    SUB("-", { target: Int, input: Int -> target - input }),
+    MUL("*", { target: Int, input: Int -> target * input }),
+    DIV(
+        "/",
+        { target: Int, input: Int ->
+            if (input == 0)
+                throw IllegalArgumentException("Can't be divided by zero.")
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
            require(input != 0) { "Can't be divided by zero." }
```

Reply:
be2c968015af3195349bd809c9cebb574a76cd71
적용하였습니다.